### PR TITLE
Add ARM docs

### PR DESCRIPTION
**Pull request by aemengo**
Add ARM docs

Resolves: https://github.com/buildpacks/docs/issues/409

### CODE REVIEW

File: content/docs/app-developer-guide/build-an-arm-app.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+On your Linux ARM machine with a [docker][docker] daemon installed, prepare the following directory tree structure.
+
+```bash
+tree ~/workspace/
+~/workspace/
+├── buildpacks
+│   └── hello-world
+└── platform
+```
```
Comment (yaelharel) [inline content/docs/app-developer-guide/build-an-arm-app.md:19]: I think that I would put it after the cloning part, and then you can show the full `tree` (how the `workspace` directory supposed to look like).

File: content/docs/app-developer-guide/build-an-arm-app.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+[[order.group]]
+id = "samples/hello-world"
+version = "0.0.1"
+optional = false
+EOF
+```
+
+### 3. Build your app
+
+Now we can build our app. For this example we'll be using the docker CLI to invoke the lifecycle directly.
```
Comment (yaelharel) [inline content/docs/app-developer-guide/build-an-arm-app.md:63]:
```suggestion
Now we can build our app. For this example we will be using the docker CLI to invoke the lifecycle directly.
```

I'm not sure but I think that in most of the docs we aren't using the long form.

File: content/docs/app-developer-guide/build-an-arm-app.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  --mount type=bind,source=~/workspace/order.toml,target=/cnb/order.toml \
+  --env CNB_PLATFORM_API=0.7 \
+  ubuntu:bionic \
+  /cnb/lifecycle/creator -log-level debug -daemon -run-image ubuntu:bionic hello-arm64
+```
+
+### 4. Run it
+
+```bash
+docker run --rm hello-arm64
```
Comment (yaelharel) [inline content/docs/app-developer-guide/build-an-arm-app.md:82]: Should we indicate in what directory we're suppose to be in this step?

File: content/docs/app-developer-guide/build-an-arm-app.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+# download and extract lifecycle
+curl -L https://github.com/buildpacks/lifecycle/releases/download/v<RELEASE-VERSION>/lifecycle-v<RELEASE-VERSION>+linux.arm64.tgz | tar xf -
+```
+
+Next we make sure that our buildpack directory is structured in a way that the lifecycle will expect.
+
+```bash
+# copy hello-world buildpack
+cp -R ~/workspace/samples/buildpacks/hello-world ~/workspace/buildpacks/hello-world/0.0.1
```
Comment (natalieparellano) [inline content/docs/app-developer-guide/build-an-arm-app.md:46]:
```suggestion
cp -R ~/workspace/samples/buildpacks/hello-world ~/workspace/buildpacks/samples_hello-world/0.0.1
```
I think this was necessary? For the lifecycle to find the directory during export... (but maybe I misremembered what the issue was)

File: content/docs/app-developer-guide/build-an-arm-app.md (hunk 1 above)
Comment (aemengo) [reply content/docs/app-developer-guide/build-an-arm-app.md:19]: If I showed the full samples repo in the diagram, the diagram would get really big and quickly get out of date... 🤷🏾‍♂️

File: content/docs/app-developer-guide/build-an-arm-app.md (hunk 4 above)
Comment (aemengo) [reply content/docs/app-developer-guide/build-an-arm-app.md]: I remember changing some metadata _inside_ a toml file, not the directory name. But either way: _fixed in latest commit_.

File: content/docs/app-developer-guide/build-an-arm-app.md (hunk 2 above)
Comment (aemengo) [reply content/docs/app-developer-guide/build-an-arm-app.md]: _Fixed in latest commit_.

**Review by jromero — APPROVED** [review]
@jkutner this seems worthy of some fanfare. 

Thank you @aemengo.

**Comment by jromero** [comment]
Should this also be waiting for 0.12.0 release of the lifecycle?

File: content/docs/app-developer-guide/build-an-arm-app.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+### 1. Prepare your working directory
+
+On your Linux ARM machine with a [docker][docker] daemon installed, prepare the following directory tree structure.
+
+```bash
+tree ~/workspace/
+~/workspace/
+├── buildpacks
+│   └── hello-world
```
Comment (natalieparellano) [inline content/docs/app-developer-guide/build-an-arm-app.md:17]:
```suggestion
│   └── samples_hello-world
│       └── 0.0.1
```

**Comment by natalieparellano** [comment]
@jromero let's point this to the `release/lifecycle/0.12.0` branch. Once this and #412 are merged into that branch, we could cut the lifecycle release.

File: content/docs/app-developer-guide/build-an-arm-app.md (hunk 5 above)
Comment (aemengo) [reply content/docs/app-developer-guide/build-an-arm-app.md]: Good catch. _Fixed in latest commit_.

**Review by natalieparellano — APPROVED** [review]

**Pull request merged**